build/cloud/07-deployment.yaml

```diff
@@ -26,6 +26,12 @@ spec:
             - containerPort: 10000
               name: cloudhub
               protocol: TCP
+            - containerPort: 10003
+              name: stream
+              protocol: TCP
+            - containerPort: 10004
+              name: tunnel
+              protocol: TCP
```

Comment:
The all containerPorts are no need in the deployment yaml I think, any ideas? @zc2638 @zhu733756

Reply:
I think on the mode of cloudcore host network, service can be disabled, and no need to add containerPorts, and in this case,  the containerports are equal to hostports. However, If we use outside Iptables manager or the cloudcore pod IP is not the host ip, we also need a service to expose it outside the cluster. Since the edges may not be able to communicate with cloudside.

Reply:
This is a test for disabled service and inner container ports:

<img width="360" alt="9ffcf0f2f81cf3f79782d0d12b86077" src="https://user-images.githubusercontent.com/38685176/146323348-a19dbf3a-fc3a-4d58-a8c5-ad6ea09ef112.png">

Reply:
yes,I have added nodeport to service.yaml.example.